test/conftest.py

```diff
@@ -16,6 +17,8 @@
 
 from . import load_fixture
 
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
This can also be changed via pytest CLI eg:
```
pytest --log-level=DEBUG
```
Not sure if we want to default to debug. It spams a lot.

Reply:
It's the kind of spam you want to see if a test breaks

Reply:
Yes, but sometimes it's in the way and too much scrolling.

Reply:
ok, will remove.